### PR TITLE
Remove useless heartbeat success log prints

### PR DESCRIPTION
**Pull request by weihubeats**
https://github.com/apache/rocketmq-clients/issues/739

### CODE REVIEW

File: golang/client.go [hunk 1]
```diff
@@ -436,7 +436,6 @@ func (cli *defaultClient) doHeartbeat(target string, request *v2.HeartbeatReques
 			Message: resp.GetStatus().GetMessage(),
 		}
 	}
-	cli.log.Infof("send heartbeat successfully, endpoints=%v", endpoints)
```
Comment (lizhanhui) [inline golang/client.go:439]: Change log level to debug instead of removal?

Comment (weihubeats) [reply golang/client.go:439]: Consider changing to debug level

**Comment by weihubeats** [comment]
@lizhanhui Is there anything else wrong with this pr?

**Comment by lizhanhui** [comment]
> @lizhanhui Is there anything else wrong with this pr?

Looks good to me.

**Comment by weihubeats** [comment]
So this mr can merge?

**Review by lizhanhui — APPROVED** [review]

**Pull request merged**